metadata-models/src/main/pegasus/com/linkedin/dataprocess/DataProcessInstanceDatasetOutput.pdl

```diff
@@ -0,0 +1,33 @@
+namespace com.linkedin.dataprocess
+
+import com.linkedin.common.DatasetUrn
+import com.linkedin.common.DatasetInputLineage
+import com.linkedin.common.Urn
+
+/**
+ * Information about the outputs of a Data process
+ */
+@Aspect = {
+  "name": "dataProcessInstanceDatasetOutput"
+}
+record DataProcessInstanceDatasetOutput {
```

Comment:
Can an instance produce a dashboard? Chart? A ML model? 

Then I would avoid hardcoding to dataset here... Instead, we can use DataProcessInstanceOutputs

Reply:
One other thought, since there are _so many_ relationships involved here.. We could just have a single "DataProcessInstanceRelationships" aspect that captures all of them... so long as the writer will typically write all of these at once...